src/Microsoft.Diagnostics.Monitoring.WebApi/Strings.resx

```diff
@@ -118,7 +118,7 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="CallstackThreadHeader" xml:space="preserve">
```

Comment:
nit: Consider that we may not want to write out an extra space on the end if the thread happens to not have a name; this will look out of place when writing it out as JSON. We may need two resource strings for this.